### ISSUE TITLE
Feature request: Support `churn --version`

### ISSUE THREAD

**Issue by etagwerker** [danmayer/churn#81]
Hey @danmayer, 

Have you thought about adding a version option? 

```
[etagwerker@bonsai metric_fu git:(master ✗)]$ bundle exec churn --version
/Users/etagwerker/Projects/fastruby/metric_fu/vendor/bundle/ruby/2.5.0/gems/churn-0.0.35/lib
unrecognized option `--version'
```

It might be useful for reporting bugs and for users to make sure that they're running the latest version.

**Comment by danmayer** [comment]
ok, sure. How is this?

check the PR... https://github.com/danmayer/churn/pull/82

```
[master][~/projects/churn] bundle exec bin/churn --version
1.0.7

[master][~/projects/churn] bundle exec bin/churn
**********************************************************************
* Revision Changes
**********************************************************************
Files
+----------------------+
| file                 |
+----------------------+
| Gemfile.lock         |
| lib/churn/version.rb |
+----------------------+



**********************************************************************
* Project Churn
**********************************************************************

```

**Comment by etagwerker** [comment]
@danmayer Thanks for the quick response! 

Just gave it a shot but I found an issue: 

```
$ churn --version
F, [2020-05-05T11:22:29.867602 #71305] FATAL -- : uninitialized constant Churn::VERSION (NameError)
/Users/etagwerker/.rvm/gems/ruby-2.6.6@fastruby/gems/churn-1.0.7/bin/churn:95:in `report_churn'
/Users/etagwerker/.rvm/gems/ruby-2.6.6@fastruby/gems/churn-1.0.7/bin/churn:108:in `run'
/Users/etagwerker/.rvm/gems/ruby-2.6.6@fastruby/gems/main-6.2.3/lib/main/program/class_methods.rb:155:in `block in run'
/Users/etagwerker/.rvm/gems/ruby-2.6.6@fastruby/gems/main-6.2.3/lib/main/program/class_methods.rb:144:in `catch'
/Users/etagwerker/.rvm/gems/ruby-2.6.6@fastruby/gems/main-6.2.3/lib/main/program/class_methods.rb:144:in `run'
/Users/etagwerker/.rvm/gems/ruby-2.6.6@fastruby/gems/main-6.2.3/lib/main/factories.rb:18:in `run'
/Users/etagwerker/.rvm/gems/ruby-2.6.6@fastruby/gems/main-6.2.3/lib/main/factories.rb:25:in `Main'
/Users/etagwerker/.rvm/gems/ruby-2.6.6@fastruby/gems/churn-1.0.7/bin/churn:14:in `<top (required)>'
/Users/etagwerker/.rvm/gems/ruby-2.6.6@fastruby/bin/churn:23:in `load'
/Users/etagwerker/.rvm/gems/ruby-2.6.6@fastruby/bin/churn:23:in `<main>'
/Users/etagwerker/.rvm/gems/ruby-2.6.6@fastruby/bin/ruby_executable_hooks:24:in `eval'
/Users/etagwerker/.rvm/gems/ruby-2.6.6@fastruby/bin/ruby_executable_hooks:24:in `<main>'
```

Here are the steps that I followed: 

```
gem install churn
churn --version
```

**Comment by etagwerker** [comment]
Hopefully #83 solves that small problem. 😄

**Comment by danmayer** [comment]
blah I tested but I guess if you bundle exec it requires all the files

**Comment by danmayer** [comment]
released `1.0.8`

**Comment by etagwerker** [comment]
@danmayer yeah, I figured that's what happened. No problem. 

Thanks for cutting that new release! 😄
